10.x/ko/eloquent-mutators.md

```
@@ -1,38 +1,38 @@
-# Eloquent: 접근자 & 캐스팅 (Eloquent: Mutators & Casting)
+# Eloquent: 뮤테이터 및 캐스팅 (Eloquent: Mutators & Casting)
 
 - [소개](#introduction)
-- [접근자와 변이자](#accessors-and-mutators)
-    - [접근자 정의하기](#defining-an-accessor)
-    - [변이자 정의하기](#defining-a-mutator)
-- [속성 캐스팅](#attribute-casting)
+- [엑세서(Accessor) 및 뮤테이터(Mutator)](#accessors-and-mutators)
+    - [엑세서 정의하기](#defining-an-accessor)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)